docs/devices_api.yml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.0.3
+
+info:
+  title: Device Connect
+  description: |
+    Device facing API for managing persistent device connections.
+
+  version: "1"
+
+  contact:
+    name: Mender Support
+    email: support@mender.io
+    url: https://mender.io/contact-us
+
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: https://hosted.mender.io/api/devices/v1/deviceconnect
+
+# Global security definitions
+security:
+  - DeviceJWT: []
+
+# Global tags
+tags:
+  - name: DevicesAPI
+
+paths:
+
+  /connect:
+    get:
+      operationId: Connect
+      summary: >
+        Connect the device and make it available to the server.
```

Comment:
Please do not introduce newlines in the summary, the rendering of the spec in our API docs breaks otherwise.